pkgs/development/python-modules/variants/default.nix

```diff
@@ -0,0 +1,29 @@
+{ buildPythonPackage
+, fetchPypi
+, pytestrunner
+, setuptools_scm
+, pytest
+, lib
+}:
+buildPythonPackage rec {
+  pname = "variants";
+  version = "0.2.0";
+
+  src = fetchPypi {
+    inherit pname version ;
+    sha256 = "511f75b4cf7483c27e4d86d9accf2b5317267900c166d17636beeed118929b90";
+  };
+
+  propagatedBuildInputs = [
+    pytestrunner
+    setuptools_scm
```

Comment:
pytestrunner and setuptools_scm should be in nativeBuildInputs, pytest should be in checkInputs

Reply:
done

Reply:
Is that always true, or just for this package @jonringer?